docs/objective_and_loss.rst

```diff
@@ -0,0 +1,43 @@
+Objective and loss functions
+============================
+
+Loss functions
+--------------
+
+All pointwise losses in JAXopt operate on a single sample. Use ``jax.vmap(loss)``
```

Comment:
It might be worth to specify what pointwise means in this context, its not clear to me